rel/commands/mana.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+release_ctl eval --mfa "Mix.Tasks.Mana.run/1" --argv -- --extra "$@"
```

Comment:
Looks like this file is missing a trailing newline. Here's a blog post on why GitHub things this is relevant https://robots.thoughtbot.com/no-newline-at-end-of-file